CHANGELOG.adoc

```diff
@@ -60,6 +60,7 @@ New default implementations are available in `DefaultToolsFactory` to ease the m
 Its single method, `getTargetObjectId`, has been moved up into the main `IRepresentation` interface.
 For representations which were *not* `ISemanticRepresentation` before, their target object is the `IEditingContext` itself if nothing more relevant is available.
 Accordingly, `IObjectSearchService` and `IIdentityService` now handle actual `IEditingContext` instances as objects.
+- https://github.com/eclipse-sirius/sirius-web/issues/3425[#3425] [diagram] `DiagramDescription` must now declare an `ArrangeLayoutDirection`.
```

Comment:
"Must" means that if you do not provide a direction it fails? If it is the case, could you provide a default value, the one existing before you set configurable the diretion?

Reply:
This breaking change was for the change in the graphql schema, I make the descritpion more precise.
I already add a default value like you suggest, it's easier to manage.

Reply:
Ok thank you